### DISCUSSION TITLE
Canvas Adjustment on different Screen Size

### DISCUSSION THREAD

**Discussion by dattap647** [fabricjs/fabric.js#9928] [General]
i Have created the the MapEditor where user can uplaod the image of map(background of canvas) and allowing to plot the object as resouce on the map . 

Suppose user  upload the high resolution image as map on the 4k screen. and another user open the same map in lower resolution screen like laptop. then the canvas is having the resolution of 4k screen. I want to makethe cavas dynamic allign and plot the object properly.

basically maintaining the aspect ratio of image and positioning of the object.
